code/scripting/api/objs/ai_helper.cpp

```diff
@@ -128,11 +82,14 @@ ADE_FUNC(turnTowardsPoint,
 	vec3d* target;
 	bool diffTurn = true;
 	vec3d* modifier = nullptr;
-	if (!ade_get_args(L, "oo|bo", l_AI_Helper.Get(&ship), l_Vector.GetPtr(&target), &diffTurn, l_Vector.GetPtr(&modifier))) {
+	float bank = 0.0f;
+
+	int argnum = ade_get_args(L, "oo|bof", l_AI_Helper.Get(&ship), l_Vector.GetPtr(&target), &diffTurn, l_Vector.GetPtr(&modifier), &bank);
+	if (argnum == 0) {
```

Comment:
Why `== 0`? There's two required arguments, right? Why not simply use the functional call itself?

Reply:
Because 0 is a special case for "error".
All other functions just do !ade_get_args, but I don't since I need to grab the actual arg number to make the last parameter truely optional

Reply:
Ah, right of course.